api/catalog/api/utils/waveform.py

```diff
@@ -128,3 +129,14 @@ def cleanup(file_name):
         os.remove(file_path)
     else:
         log.info("File not found, nothing deleted")
+
+
+def generate_peaks(audio) -> List[float]:
```

Comment:
As of Python 3.10, we can use `list[float]` as the type without needing to import `List` from `typing`.

Reply:
Ah, I thought so but then I forgot to go back and change it!